modules/account-lib/src/coin/avaxc/utils.ts

```diff
@@ -50,7 +50,14 @@ export function isValidEthAddress(address: string): boolean {
  * @returns {boolean} - the validation result
  */
 export function isValidEthPrivateKey(privateKey: string): boolean {
+  if (privateKey.length != 64) {
```

Comment:
```suggestion
  if (privateKey.length !== 64) {
```
enforces stricter type check and also suppresses eslint complaint